### PR TITLE
Add browser kit testing

### PR DESCRIPTION
**Pull request by sdrafahl**
https://trello.com/c/pj0z2ZOW/4-setup-frontend-testing

### CODE REVIEW

File: browserTest/test.js [hunk 1]
```diff
@@ -0,0 +1,20 @@
+let webdriver = require('selenium-webdriver');
```
Comment (jd1388) [inline browserTest/test.js]: Did `import` statements not work?

File: browserTest/test.js [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+let test = require('selenium-webdriver/testing');
+let assert = require('assert');
+
+let driver = null;
+
+test.describe('webdriverjs test', () => {
```
Comment (jd1388) [inline browserTest/test.js]: Is `test.describe` required for the selenium driver or for Mocha?

File: browserTest/test.js [hunk 3]
```diff
@@ -0,0 +1,20 @@
...
+    test.before((done) => {
```
Comment (jd1388) [inline browserTest/test.js]: Same as above

File: browserTest/test.js [hunk 4]
```diff
@@ -0,0 +1,20 @@
...
+        driver = new webdriver.Builder().forBrowser('chrome').build();
+        done();
+    });
+
+    test.it('Login Should Exist', (done) => {
```
Comment (jd1388) [inline browserTest/test.js]: Same as above

File: browserTest/test.js (hunk 1 above)
Comment (sdrafahl) [reply browserTest/test.js]: Not supported by Node.js, one of the few things not supported by Node.js in ES6.

**Pull request merged**